_drafts/asking-for-help.md

```diff
@@ -0,0 +1,40 @@
+As some of us find ourselves with extra time on our hands in the COVID-19 era many at Fred Hutch are looking to sharpen their programming skills. Luckily, we have some amazing opportunities and resources to learn digitally through The Coop, TDS-IRC, and FredHutch.io. A natural consequence of more people learning to code is that we've seen an increase in messages related to looking for coding help via slack channels, community groups, and personal messages to folks like Kate and myself. We've noticed a stark difference in how seasoned and newer programmers will structure their messages seeking help. Someone newer to programming may send a message that goes something like this: `Hi, I can't install tidyverse. Can you please help?`. While there is nothing technically wrong with asking for help this way - it will take a few back and forth messages, possibly running some code again, to diagnose the issue. In this post we will go over some norms and best-practices for asking for coding help through slack, email, or online forum.
```

Comment:
Suggested revisions:

As some of us find ourselves with extra time on our hands in the COVID-19 era, many at Fred Hutch are looking to sharpen their programming skills. Luckily, we have some amazing opportunities and resources to learn digitally through groups like the Coop/fredhutch.io and TDS-IRC. A natural consequence of more people learning to code is an increase in messages to Slack channels, community groups, and personal messages asking for assistance with coding and software. At first glance, many of these messages seem straightforward: `Hi, I can't install tidyverse. Can you please help?`. This is a reasonable and polite request, but is actually difficult for us to answer. It will take a few back and forth messages, possibly running some code again, to diagnose the issue. How can you make this process run more smoothly? In this post we will go over some norms and best-practices for asking for coding help through slack, email, or online forum.